tech-far-hub/content/get-started/competition-requirements.mdx

```diff
@@ -0,0 +1,20 @@
+---
+slug: competition-requirements
+heading: Competition Requirements
+seo_title: Competition Requirement
+tags: [ tag1, tag2, tag3 ]
+lifecycle_stage: 2
```

Comment:
Saw this being used in the DeepPick utility, curious to find out what it's being used for...